CHANGES.md

```diff
@@ -13,6 +11,18 @@ Format of the entries must be.
 * Entry two with no-newlines. (DCOS_OSS_JIRA_2)
 ```
 
+## DC/OS 1.10.9
```

Comment:
The patch was included into the last 1.10.8 train.

https://github.com/dcos/dcos/pull/3081/commits/aa7142726a2a3d16937f17e1170889b0a81957bb